src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2581,6 +2581,28 @@ public function escapeWhenCastingToString($escape = true)
         return $this;
     }
 
+    /**
+     * Get the properties for serialization.
+     *
+     * @return array
```

Comment:
You can narrow this further down:

```suggestion
     * @return array<int, string>
```
or

```suggestion
     * @return string[]
```
or

```suggestion
     * @return list<string>
```

Reply:
Yeah aware, thanks. But I checked the class and it's not yet used there. Leaving it to Taylor.